tests/examples/epcisQueryDocHavingEventWithIgnoreFields.hashes

```diff
@@ -0,0 +1,2 @@
+ni:///sha-256;b3d481c5590757ab8361a6cb0e924820feb3b61eb568e7d7703f21a96f76f51d?ver=CBV2.0
```

Comment:
Considering Sven's comment:
https://github.com/RalphTro/epcis-event-hash-generator/issues/109#issuecomment-1580864902

I think it would be a really good idea to add the ignored fields into the NI.

In the current PR state, you are not able to check if an event (only the event object) equals its hash (NI) since neither contains the list of fields to be excluded from the hash.